.circleci/continue_config.yml

```diff
@@ -57,7 +57,7 @@ executors:
       - image: prestocpp/prestocpp-avx-centos:root-20230613
     resource_class: 2xlarge
     environment:
-      MAVEN_OPTS: "-Xmx4G -XX:+ExitOnOutOfMemoryError"
+      MAVEN_OPTS: "-Xmx4G -XX:+ExitOnOutOfMemoryError -XX:CompressedClassSpaceSize=3g"
```

Comment:
Why did you make this change ? It might be better to make an individual PR for this change.

Reply:
The following error was seen with CircleCI job `linux-presto-e2e-tests` and this change was an attempt to fix it: 
```
[ERROR] java.lang.OutOfMemoryError: Compressed class space
[ERROR] Terminating due to java.lang.OutOfMemoryError: Compressed class space
....
[ERROR] com.facebook.presto.nativeworker.TestPrestoNativeWindowQueries
[ERROR] org.apache.maven.surefire.booter.SurefireBooterForkException: ExecutionException The forked VM terminated without properly saying goodbye. VM crash or System.exit called?
```
Reverting it, this change did not fix the error.